src/components/TodoList/todolist.scss

```diff
@@ -0,0 +1,29 @@
+.todo-list {
+  width: 500px;
+  background-color: hsl(237deg, 14%, 26%);
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+
+  &__grid{
+    max-width: 100%;
+    display: grid;
+    grid-template-columns: 6% 88% 6%;

```

Comment:
In my opinion, `fr` is better than `%`, the why is because `%` doesn't consider `grid-gap` in its division, it will sum overflowing the width defined to the grid element.

consider the image below:
![Screen Shot 2022-09-20 at 10 49 39 AM](https://user-images.githubusercontent.com/2808008/191275514-2d33e10e-b833-483f-9737-4b1852cae3cd.png)

they have the same width for the grid element but one is considering only the free space (fr) and the other don't.

```css
.root {
  width: 500px;
  display: grid;
  grid-gap: 20px;
  grid-template-columns: ; // The first uses `fr` another uses `%`
}
```

In summary, the columns will be divided wrongly (if [grid-gap](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) is used).